### PR TITLE
Doc14main

### PR DESCRIPTION
**Pull request by 0815Creeper**

### CODE REVIEW

**Comment by 0815Creeper** [comment]
doc action is expected to fail. Mergable if examples pass

File: .github/workflows/CrossChecks.yml [hunk 1]
```diff
@@ -23,18 +23,18 @@ jobs:
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - name: Check out repository 
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
 
       # Set up Julia
       - name: "Set up Julia"
-        uses: julia-actions/setup-julia@v1
+        uses: julia-actions/setup-julia@latest
```
Comment (ThummeTo) [inline .github/workflows/CrossChecks.yml:30]: why this? couldn't there be breaking changes in the actions?

File: .github/workflows/Documentation.yml [hunk 2]
```diff
@@ -23,12 +23,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: "Check out repository"
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
       - name: "Set up Julia"
-        uses: julia-actions/setup-julia@v1
+        uses: julia-actions/setup-julia@latest
```
Comment (ThummeTo) [inline .github/workflows/Documentation.yml:29]: `same here, isn't it better to use a fixed version (the current one for now)?

File: .github/workflows/Documentation.yml [hunk 3]
```diff
@@ -23,12 +23,12 @@ jobs:
...
         with:
-          version: '1'
+          version: '1.10'
```
Comment (ThummeTo) [inline .github/workflows/Documentation.yml:31]: here, the other way round - we fix Julia version 1.10 (but 1.11 is on its way)

**Comment by 0815Creeper** [comment]
changed as requested

**Pull request merged**